src/Tests/Microsoft.NET.TestFramework/Commands/ComposeStoreCommand.cs

```diff
@@ -20,24 +21,23 @@ public ComposeStoreCommand(ITestOutputHelper log, string projectPath, string rel
         }
 
         public override DirectoryInfo GetOutputDirectory(string targetFramework = "netcoreapp1.0", string configuration = "Debug", string runtimeIdentifier = "")
-        {
-            string output = Path.Combine(ProjectRootPath, "bin", BuildRelativeOutputPath(targetFramework, configuration, runtimeIdentifier));
-            return new DirectoryInfo(output);
-        }
-
-        public string GetPublishedAppPath(string appName)
-        {
-            return Path.Combine(GetOutputDirectory().FullName, $"{appName}.dll");
-        }
-
-        private string BuildRelativeOutputPath(string targetFramework, string configuration, string runtimeIdentifier)
         {
             if (runtimeIdentifier.Length == 0)
             {
                 runtimeIdentifier = RuntimeInformation.RuntimeIdentifier;
             }
             string arch = runtimeIdentifier.Substring(runtimeIdentifier.LastIndexOf("-") + 1);
-            return Path.Combine(configuration, arch, targetFramework, PublishSubfolderName);
+            string relativeToBaseOutputPath = Path.Combine(configuration, arch, targetFramework, runtimeIdentifier, PublishSubfolderName);
```

Comment:
This is to match the MSBuild output path logic in the common targets.

Reply:
These are helper methods for the ComposeStoreCommand (which is for `dotnet compose) which as far as I can tell are never used.  Since we don't need them, it's better to delete them than to try to correct them.